docs/docs/agents/memory.md

```diff
@@ -149,6 +149,75 @@ agent = create_react_agent(
 
 To learn more about using `pre_model_hook` for managing message history, see this [how-to guide](../how-tos/create-react-agent-manage-message-history.ipynb)
 
+### Read in tools { #read-short-term }
```

Comment:
The split is odd with read not being here, but write being here

Reply:
Ideally, we would have the info in both places through a reusable content chunk (to minimize diverging content). How about something like this instead:

```
LangGraph allows agent to access its short-term memory (state) inside the tools:

<ADD STATE CODE SAMPLE HERE>

See the [Context](https://langgraph-docs-preview-mdi31bdl7-langchain.vercel.app/agents/context/#__tabbed_2_2) guide for more information.
```

Reply:
sounds good!